### ISSUE TITLE
Support for .html not available as answer, rather than only .png

### ISSUE THREAD

**Issue by michaelsjackson** [c-okelly/org_to_anki#72]
In contrast to .pdf .html could work, tested also this, but also .html as answer did not work here.  Could be another useful alternative to .png (or images) in certain situations, meaning where you have already some .html but no .png, so why not just using that .html directly.

**Comment by c-okelly** [comment]
Can you expand on the example? I'm not exactly sure what you mean?

Like a secondary .html file that is used is just inserted as is?

**Comment by michaelsjackson** [comment]
Yes, exactly, as you put the path to .png, we would put the path to some .html file on our harddisk, same as pdf example would be a prepared .html with a few sentences only inside.

One example:  Those .html files could contain source codes, converted from source codes via highlight for example, just as one use case scenario.

**Comment by c-okelly** [comment]
Ye it's an interesting idea but I really don't think there is much demand for something like that.

Also the add-on already support source code with syntax highlighting

**Comment by michaelsjackson** [comment]
Well, I do not use it for source codes.  :smile:  It should be just an example so many people understand where it could be used.  I create my own language definitions, on the fly, just adding a list of "important" words, so those are shown in a different color or bold, just to emphasize them.  If you want to convert quickly a huge list of sentences, for example books  :smile_cat: this technique can be useful, even not knowing what the book is about beforehand, you would just do some pre analysis on most used words, most used word pairs, filtering out disturbing stopwords, plus use automatically generated noun lists, a combination of those, then take e.g. head 50 of those words, just as one technique.   We can combine tools freely, even if it was developed originally for source code highlighting, we can highlight any other text as well.

Regarding not much demand, is a weak thinking style, but no problem, whoever needs it so hardly can develop and use it, the idea is here now, anyone can read and think about it.  Thanks for responding.

Those important words one could not only highlight but also cloze delete!

**Comment by c-okelly** [comment]
> Regarding not much demand, is a weak thinking style, but no problem, whoever needs it so hardly can develop and use it, the idea is here now, anyone can read and think about it. Thanks for responding.

To be more clear on what I really mean is that without a few people asking for something I do find it a bit hard to understand exactly what their usecase is. Thus is can be hard to design acceptance criteria to meet them. I general the conversation is a lot easier with some who is technical as they can express more clearly what they want. 

For the example of the html files a few question would come to to mind straight away.

1. Would Anki support a full html page being embedded eg `<html>.......</html>`
2. Should I expect a full html page from the user
3. If i just support sinppets instead because anki does support a full page do I warn users that they are uploading won't work or do I try and parse out a snippet from what they have provided
4. etc

**Comment by michaelsjackson** [comment]
1.  This is not necessary, just take the content of body, ignoring rest.
2.  No, only what is output of highlight.  Preprocessed according to 1.

Important is to find a way which will work always in Anki.

Probably I should check the requirements of Anki first.
